frontend/next.config.js

```diff
@@ -1,57 +1,19 @@
-// next.config.js
-//
-// NOTE: Make sure you update `src/__mocks__/next/config.js` when change this file!
-//
-const { DATA_SOURCE_TYPE } = require("./src/libraries/consts.js");
-
-const defaultWampNearExplorerUrl = "ws://localhost:8080/ws";
-
-let nearNetworks;
-if (process.env.NEAR_NETWORKS) {
-  nearNetworks = JSON.parse(process.env.NEAR_NETWORKS);
-} else {
-  nearNetworks = [
-    {
-      name: "localhostnet",
-      explorerLink: "http://localhost:3000",
-      aliases: ["localhost:3000", "localhost", "127.0.0.1", "127.0.0.1:3000"],
-      nearWalletProfilePrefix: "https://wallet.near.org/profile",
-    },
-  ];
-}
-const nearNetworkAliases = {};
-for (const nearNetwork of nearNetworks) {
-  for (const alias of nearNetwork.aliases) {
-    nearNetworkAliases[alias] = nearNetwork;
-  }
+// Jest registers ts-node for us, double run of ts-node/register will break code
```

Comment:
@frol @shelegdmitriy 
JFYI that's a very tricky point with double registration of `ts-node`, if you ever stumble upon `TS Error: Emit skipped` or types mismatch in properly typed file - let me know.